addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -172,6 +172,7 @@ Object.assign(ThreadView, {
         hasSquashCloseMessages: false,
         haveMessagesMarkAsReadIcon: false,
         haveMessagesReplyIcon: false,
+        haveMessagesUnfollowIcon: false,
```

Comment:
This is just a forward to message list, it should not have a default value here if message list already has one.

Reply:
We have `ThreadView` form chatWindow, chatter and discuss. And here we are passing props `haveMessagesUnfollowIcon` only from the discuss, thus we need to define default value.

Reply:
So, handle it with `optional : true`.